### PR TITLE
Feature 2 red

### PR DESCRIPTION
**Pull request by orhanna14**
The goal of feature 2 is to output numbers above each column, and letters to the left of each row of the grid. 

This adds a failing test for that. The structure is identical to the first test.

### CODE REVIEW

File: spec/game_runner_spec.rb [hunk 1]
```diff
@@ -20,5 +20,25 @@
 
       expect(stdout.string).to eq(grid_template)
```
Comment (orhanna14) [inline spec/game_runner_spec.rb]: I'm thinking I'll need to change this to `expect(stdout.string).to include(grid_template)`

File: README.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ It's pretty easy to throw a script into the middle of an object. That isn't this
 * Don't use local variables, make methods instead that capture that logic (and memoize if necessary)
 * Name your methods well, and notice when the names start to group together. It's probably a missing class.
 * Arguments to methods may be a sign of a missing object. This is particularly true when you need to pass the same argument around to several methods
-* The inside of loops are also candidates for seprate objects, since they have different state than outside the container
+* The inside of loops are also candidates for seperate objects, since they have different state than outside the container
```
Comment (baccigalupi) [inline README.md]: Can you put this in a separate PR and make it against the thoughtbot project instead?

**Review by baccigalupi — APPROVED** [review]
README changes shouldn't be in this. Please move it to a PR against the main repo.

Otherwise the change looks good!

File: README.md (hunk 2 above)
Comment (orhanna14) [reply README.md]: Yep will do!

**Review by baccigalupi — CHANGES_REQUESTED** [review]
Oops. Sorry this shouldn't be a new spec. The existing one should be modified

**Review by baccigalupi — APPROVED** [review]
